ci/includes/pinned-tools.sh

```diff
@@ -24,10 +24,15 @@ function getUnityDir() {
 }
 
 UNITY_DIR="$(getUnityDir)"
+export UNITY_HOME="${UNITY_DIR}"
```

Comment:
Why is this overwriting the user's UNITY_HOME?

Reply:
because if it's not set, the code generator tests fail

Reply:
This will go away with Jared's PR anyway, I believe: https://github.com/spatialos/UnityGDK/pull/57

Reply:
This might be overwriting it with a slightly different version. Can we change the name of our own UNITY_HOME to something else that can be exported for this specific session.

Reply:
as mentioned above, I would prefer to leave it in for now just so that CI passes. @jared-improbable this line can be removed when your PR goes in, right?

Reply:
Yep, this line isn't long for this world.